dali/base/dasds.cpp

```diff
@@ -5690,59 +5693,73 @@ class CStoreHelper : implements IStoreHelper, public CInterface
         return res;
     }
 
-    unsigned saveStoreToFile(const IPropertyTree *root, IFileIO *iFileIOTmpStore, StoreFormat format)
+    IFile *saveStoreToFile(const IPropertyTree *root, const StoreFormat format, unsigned &crc)
```

Comment:
style: generally, scalars are not declared as const, since they are passed by value it has little benefit